lisa/sut_orchestrator/azure/features.py

```diff
@@ -556,9 +570,10 @@ def _initialize(self, *args: Any, **kwargs: Any) -> None:
         self._is_nvidia = True
 
     def _install_driver_using_platform_feature(self) -> None:
+        # https://learn.microsoft.com/en-us/azure/virtual-machines/extensions/hpccompute-gpu-linux
         supported_versions: Dict[Any, List[str]] = {
```

Comment:
will it impact all of gpu test cases?

Reply:
As per documentation GPU installation using extension is only supported in these. The support for the older distro's are probably removed now.

Reply:
Do you want me to revert and keep it as the old values itself?